README.md

```diff
@@ -149,10 +149,12 @@ Running on Bluemix? You can initialize Cloudant directly from the `VCAP_SERVICES
 
 ~~~ js
 var Cloudant = require('cloudant');
-var cloudant = Cloudant({instanceName: 'foo', vcapServices: JSON.parse(process.env.VCAP_SERVICES)});
+var cloudant = Cloudant({ vcapInstanceName: 'foo', vcapServices: JSON.parse(process.env.VCAP_SERVICES) });
```

Comment:
I see that `instanceName` can be still be used as an alias for `vcapInstanceName`, this should probably be mentioned somewhere. Initially when I saw this in the README I assumed it was a breaking change.

Reply:
Added note about alias `instanceName`.